packages/block-library/src/classic/editor.scss

```diff
@@ -112,7 +112,7 @@
 		margin: 15px auto;
 		outline: 0;
 		cursor: default;
-		background-image: url(/wp-includes/js/tinymce/skins/wordpress/images/more-2x.png);
+		//background-image: url(/wp-includes/js/tinymce/skins/wordpress/images/more-2x.png);
```

Comment:
This is a bug actually, because this CSS is supposed to be usable in npm packages too and this image is not available. Can we avoid using an image. cc @draganescu

Reply:
Hi @youknowriad we've been through this here #14173 and while it looks bad it is needed more or less. The good part is that it works specifically for the classic editor, if we don't have it available this Gutenberg block is useless, which makes the missing image a non problem.

Reply:
Is there a way to recreate something close to this design with CSS/HTML?

Reply:
There's also base64 data URIs as an option:

```
background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAADtgAAAAoBAMAAAA86gLBAAAAJFBMVEVMaXG7u7vBwcHDw8POzs68vLzGxsbMzMy+vr7AwMDQ0NDGxsYKLGzpAAAADHRSTlMA///zWf+/f///TMxNVGuqAAABwklEQVR4Ae3dMXLaQBTH4bfj8UCpx8hq0vgKvgFNemhT6Qo6gg6R+0ZahM2QLmyBJ99XWP9V5+o3jIUcLQEAAAAAAAAAAAAAAAAAAAAAAABQ8j0WL9lfTtlt18uNXAUA8O/KVtfa1tdcrOdSh9gCQAMlh1hMNbZZ1bsrsQWABsrhLRbz7z5in/32UbfUMUbkMQCAh5RfGYv82UdMdZ6HS2wjT2ILAI8r3XmM2B3WvM59vfO2xXYW2yYAENuPU8S+X/N67mKxzy225yaxBQCxLV392UdcvwV0jPVUj98ntkBWT7C7+9u2/V/vGtvXIWJ6/4rtbottWa6Ri0NUT/u72LYttrb97LHdvUXMxxrb8TO2W2TF1rYbbLG1bbGNjMi4+2Sbi1FsbbvNFlvbFtt5fDnE3d9sP1/XeIyV2Nr2U2/guZUuptNrH/dPI9eLB6SaAEBs6wPJf3/PNk9tYgsAYrv/8TFuzx/fvkFqGtrEFgDEdpcZUb7ejXy6ntrEFgDENvL6gsas4vbdyKt4DACI7TxElJv/Z7udpqFNbAFAbKduy2uU2trttM/x28UWAAAAAAAAAAAAAAAAAAAAAAAAAADgDyPwGmGTCZp7AAAAAElFTkSuQmCC);
```

And a bit smaller if the image could be just the "More" text, with the dash applied otherwise via something like CSS `border`:

```
background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEgAAAATCAMAAAAEertBAAAAJFBMVEUAAADOzs7GxsbNzc3Dw8PQ0NDGxsa8vLy+vr7AwMC7u7vBwcHdVzDyAAAAB3RSTlMAWb9/80zMprLfcwAAAO1JREFUeNqFlFkSwyAMQ9laU+X+9+2Q8UKIEvyR1pb1igk0ATiSRwOQPcMZoUIjzwVrWlujTWBBfBxUrLNOIODFBwY6IAbq+ImCuiPDN1WKFywgn3ntWUEVUZWbD5A7CElQbLLDtK6f3Nc5qEFMz6aJLUipV59wUIo9cG09FNl9qlKQoOhkHFQXEPyLxWkcjWKTBagwUBgZSB/juVmRRUkPoD6kNtLNHsUVYXuks8mo7t6aZxwU7253jiLloI5vQ5m1l5NdNWegCjOC3TVJq8ZBc/fT7ed3zcNB3ZfP/49C8+E4qPLVBia0+N0Z9AdQQBQ2RTRZ7gAAAABJRU5ErkJggg==);
```